### DISCUSSION TITLE
Highlighting feature

### DISCUSSION THREAD

**Discussion by Blipz** [Azgaar/Armoria#114] [Ideas]
When working with multiple identical elements, it would be useful to be able to visually link editor sections with corresponding elements on the shield, maybe something like that:

![highlight](https://github.com/Azgaar/Armoria/assets/21015346/70888c63-cdaf-4294-b49a-33737d33635f)

The highlighting could be triggered when clicking:
- on the element on the shield;
- on a button in the editor section.

**Comment by Azgaar** [comment]
Sounds interesting. I would prefer an on hover highlight of the element on the shield, the line in editor is already highlighted.

**Comment by Blipz** [comment]
Still need to somewhat highlight the line when clicking/hovering the element on the shield, though.
I was a bit wary about suggesting hovering, if there were accessibility concerns for tablet/phone (?) users, but I guess that's not a critical feature.

**Comment by Blipz** [comment]
Done here: 8b8f3028e6e900359aec0f11c4fc4e36e8b1c7e7
